README.md

```diff
@@ -1,99 +1,63 @@
+Primary target is supporting Red Hat's assemblies of Backstage (specifically supporting [dynamic-plugins](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/dynamic-plugins.md)) on OpenShift, such as [Janus-IDP](https://janus-idp.io/) and RHDH but is flexible enough to install any correct Backstage instance (specific Kubernetes deployment with accompanying resources, see [Configuration](docs/configuration.md))
+The Operator should provide clear and flexible configuration options to satisfy wide range of expectations: from "default (no)configuration for quick start" to "higly customized configuration for production".
+[More documentation...](#More documentation)
 
 ## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
+You’ll need a Kubernetes cluster to run against. You can use [Minikube](https://minikube.sigs.k8s.io/docs/) or [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
 **Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
 
-### Running on the cluster
-1. Install Instances of Custom Resources:
-```sh
-kubectl apply -f config/samples/
-```
-2. Build and push your image to the location specified by `IMG`:
-```sh
-make docker-build docker-push IMG=<some-registry>/backstage-operator:tag
-```
-3. Deploy the controller to the cluster with the image specified by `IMG`:
-```sh
-make deploy IMG=<some-registry>/backstage-operator:tag
-```
-### Uninstall CRDs
-To delete the CRDs from the cluster:
-```sh
-make uninstall
-```
-### Undeploy controller
-UnDeploy the controller from the cluster:
-```sh
-make undeploy
-```
-## Build and Deploy with OLM
-1. To build operator, bundle and catalog images:
+To test it on minikube from the source code:
+
+You need ready to use **kubectl** and **minikube** [tools](https://kubernetes.io/docs/tasks/tools/) as a prerequisite.
+
+1.  Get your copy of Operator from GitHub: 
 ```sh
-make release-build
+git clone https://github.com/janus-idp/operator
 ```
-2. To push operator, bundle and catalog images to the registry:
+2. Deploy Operator on the minikube cluster:
 ```sh
-make release-push
+cd <your-janus-idp-operator-project-dir>
+make deploy
 ```
-3. To deploy or update catalog source:
+you can check if the Operator pod is up by running 
 ```sh
-make catalog-update
+kubectl get pods -n backstage-system
+It should be something like:
+NAME                                           READY   STATUS    RESTARTS   AGE
+backstage-controller-manager-cfc44bdfd-xzk8g   2/2     Running   0          32s
 ```
-4. To deloy the operator with OLM
+3. Create Backstage Custom resource on some namespace (make sure this namespace exists)
 ```sh
-make deploy-olm
+kubectl -n <your-namespace> apply -f examples/bs1.yaml
```

Comment:
why is `--namespace=backstage` a forced default and if I need to deploy to another namespace, how can I do that?

Reply:
> why is `--namespace=backstage` a forced default and if I need to deploy to another namespace, how can I do that?

Yes, that was also my point above. This file was our very first sample CR and hasn't changed since.
We'll need to remove the namespace from it: https://github.com/janus-idp/operator/blob/main/examples/bs1.yaml#L5

Reply:
fixed